ztp/policygenerator/policyGen/policyBuilder.go

```diff
@@ -86,6 +86,16 @@ func (pbuilder *PolicyBuilder) Build(policyGenTemp utils.PolicyGenTemplate) (map
 					if err != nil {
 						return policies, err
 					}
+					//set to default remediationAction or user set from PGT
+					remediationActionVal := policyGenTemp.Spec.RemediationAction
+					if sFile.RemediationAction != utils.UnsetStringValue {
+						// sourceFile remediationAction overrides the default remediationAction
+						remediationActionVal = sFile.RemediationAction
+					}
+
+					acmPolicy.Spec.RemediationAction = remediationActionVal
+					acmPolicy.Spec.PolicyTemplates[0].ObjDef.Spec.RemediationAction = remediationActionVal
+
```

Comment:
I am thinking can me make these into a function that be called in function createAcmPolicy() like the function BuildObjectTemplate for complianceType? I feel it's more clean and make code consitstent.